diff --git a/setup.py b/setup.py
index 2c4b558..0abff32 100755
--- a/setup.py
+++ b/setup.py
@@ -17,13 +17,12 @@ with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'README.rst')
 
 setup(
     name='afew',
-    use_scm_version={'write_to': 'afew/version.py'},
+    version='3.0.1',
     description="An initial tagging script for notmuch mail",
     url="https://github.com/afewmail/afew",
     license="ISC",
     long_description=long_description,
     long_description_content_type="text/x-rst",
-    setup_requires=['setuptools_scm'],
     packages=find_packages(),
     test_suite='afew.tests',
     package_data={
